style.css

```diff
@@ -0,0 +1,201 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding-top: 20px;
+  padding-left: 478px;
+
```

Comment:
Please, remove all extra blank lines.

Reply:
Fixed. I deleted all extra spaces in my code